xfields/beam_elements/beambeam_src/beambeam3d.h

```diff
@@ -6,23 +6,34 @@
 #ifndef XFIELDS_BEAMBEAM3D_H
 #define XFIELDS_BEAMBEAM3D_H
 
+//#if !defined(dump_path)
+//    #define dump_path "/Users/pkicsiny/phd/cern/xsuite/outputs/n74c" 
+//#endif
 
 /*gpufun*/
 void synchrobeam_kick(
-        BeamBeamBiGaussian3DData el, const int i_slice,
+        BeamBeamBiGaussian3DData el, LocalParticle *part, 
+        const int64_t flag_beamstrahlung, const int64_t flag_luminosity,
+        BeamBeamBiGaussian3DRecordData beamstrahlung_record, RecordIndex beamstrahlung_table_index, BeamstrahlungTableData beamstrahlung_table,
+        BeamBeamBiGaussian3DRecordData luminosity_record, RecordIndex luminosity_table_index, LuminosityTableData luminosity_table,
+        const int i_slice,
         double const q0, double const p0c,
         double* x_star,
         double* px_star,
         double* y_star,
         double* py_star,
         double* zeta_star,
-        double* pzeta_star){
+        double* pzeta_star, double const tan_phi){
```

Comment:
Why do you pass tan_phi? I can be extracted from el if needed.

Reply:
just for debugging. I will remove it.

Reply:
removed